### PR TITLE
Implement certificate pinning for Android SDK

### PR DESCRIPTION
**Pull request by ericmann**

### CODE REVIEW

**Review by m4dc4p — APPROVED** [review]
Nice work!

**Review by lkwdwrd — APPROVED** [review]
The logic seems sound. Looks good to me 👍 

Nice job!

File: README.md [hunk 1]
```diff
@@ -123,6 +123,61 @@ called on the same background thread used for E3DB interactions.
 E3DB operations do not have timeouts defined -- you will have to
 manage those within your own application.
 
+# Certificate Pinning
```
Comment (gstro) [inline README.md:126]: Good call adding this to the README.

**Review by gstro — APPROVED** [review]
💯 Awesome job!

**Pull request merged**